tests/core/creators/manual/detached.html

```diff
@@ -0,0 +1,66 @@
+<button id="attachButton">Attach editor to parent</button>
+
+<div>
+	<div id="editor1">
+		<p>Lorem ipsum dolor sit amet 1.</p>
+	</div>
+</div>
+
+<button id="attachButton2">Attach second editor to parent</button>
+<div>
+	<div id="editor2">
+		<p>Lorem ipsum dolor sit amet 2.</p>
+	</div>
+</div>
+
+<script>
+	var editorElement = document.getElementById( 'editor1' ),
+		attachButton = CKEDITOR.document.getById( 'attachButton' ),
+		editorParent = editorElement.parentNode;
+
+	editorParent.removeChild( editorElement );
+
+	CKEDITOR.replace( editorElement, {
+		delayDetached: true,
+		on: {
+			instanceReady: function() {
+				this.setData( 'Editor created' );
+			}
+		}
+	} );
+
+	attachButton.on( 'click', function() {
+		editorParent.appendChild( editorElement );
+	} );
+
+	//
+	// SECOND EDITOR WITH CALLBACK
+	//
+	var secondEditor = document.getElementById( 'editor2' ),
+		secondAttachButton = CKEDITOR.document.getById( 'attachButton2' ),
+		secondEditorParent = secondEditor.parentNode,
+		editorCreationCallback = null;
+
+	secondEditorParent.removeChild( secondEditor );
+
+	CKEDITOR.replace( secondEditor, {
+		delayDetached: true,
+		delayDetachedFrequency: 50,
+		registerCallback: RegisterCallback,
+		on: {
+			instanceReady: function() {
+				this.setData( 'Editor created from config callback' );
+			}
+		}
```

Comment:
This code is not checked now, so we should either remove it or add to test verification steps.

Reply:
Removed.